ecommerce/serializers.py

```diff
@@ -309,6 +357,14 @@ def update(self, instance, validated_data):
             with transaction.atomic():
                 basket.basketitems.all().delete()
                 basket.couponselection_set.all().delete()
+
+        if data_consents is not None:
+            sign_date = datetime.now(tz=pytz.UTC)
+            for consent_id in data_consents:
```

Comment:
Can we replace this for loop with something like `models.DataConsentUser.objects.filter(id__in=data_consents).update(consent_date=sign_date)`